### PR TITLE
Create a website based on Jekyll

### PR DESCRIPTION
**Pull request by kdungs**
Integrates nicely with GH pages and provides everything we need. We can (ab)use the posts for the history of events and have pages for material and general information.

### CODE REVIEW

**Comment by kdungs** [comment]
If you want to see what it looks like, go here: http://kdungs.github.io/starterkit/

File: index.html [hunk 1]
```diff
@@ -0,0 +1,12 @@
+---
+layout: default
+---
+
+<div class="home">
+
+  <img src="img/starterkit.png" alt="" />
+  <p>
+    The LHCb Starterkit is an annual workshop for Master students and early PhD students. It covers basic tools such as Bash, Git, and Python as well as LHCb specific software projects.
```
Comment (betatim) [inline index.html]: I'd remove the annual. Presumably we put the June event in the starterkit brand as well? Otherwise we need a webpage for that too :-/

**Comment by betatim** [comment]
Can we have the list of previous starterkits somewhere more prominent in the main page? It took me a while to figure out that I should click "Archive" to see them.

**Comment by betatim** [comment]
I like the look :rainbow:

**Comment by betatim** [comment]
We haz webpage.

:rocket: and :beers: nice work!

**Pull request merged**